reana_client/cli/outputs.py

```diff
@@ -162,7 +162,12 @@ def outputs_download(ctx, user, organization, workflow, file_, output_folder):
                                                              file_name)
             logging.info('{0} binary file downloaded ... writing to {1}'.
                          format(file_name, output_folder))
-            with open(os.path.join(output_folder, file_name), 'wb') as f:
+
+            outputs_file_path = os.path.join(output_folder, file_name)
+            if not os.path.exists(os.path.dirname(outputs_file_path)):
+                    os.makedirs(os.path.dirname(outputs_file_path))
```

Comment:
`exist_ok` is by default `False`, so if a user attempts to re-download a file an exception would be raised... Shall we create some nice message back to the user?  

(We could add CLI a flag later to let the user govern this behaviour, but not right now...)